Added D-Spy, Maniatic Launcher, Touché, Spelling Bee, Typography, SysD Manager, Sysprof (#357)
myfreeweb/awesome-gtk: README.md

  - [Emulators and Compatibility Layers](#emulators-and-compatibility-layers)
- [Development](#development)
  - [Containers](#containers)

>   - [Development tools](#development-tools)

Reviewer: ```suggestion
  - [Development Tools](#development-tools)
```

- [Toolbx Tuner](https://github.com/13hannes11/toolbx-tuner) - [toolbx](https://containertoolbx.org) containers manager `#rust` `#gtk4` `#libadwaita`.
- [Virtual Machine Manager (virt-manager)](https://github.com/virt-manager/virt-manager) - Graphical tool for managing virtual machines such as QEMU/KVM, Xen, LXC `#python` `#gtk3`.

> ### Development tools

Reviewer: ```suggestion
### Development Tools

```

- [Glade](https://glade.gnome.org) - RAD tool to enable quick & easy development of user interfaces for the GTK toolkit and the GNOME desktop `#c` `#gtk3`.
- [Cambalache](https://gitlab.gnome.org/jpu/cambalache) - RAD tool for Gtk 4 and 3 with a clear MVC design and data model first philosophy `#python` `#gtk3`.
- [Gradience](https://github.com/GradienceTeam/Gradience) - Libadwaita applications customizer `#python` `#gtk4` `#libadwaita`.

> - [Typography](https://flathub.org/apps/org.gnome.design.Typography) - Tool working with the GNOME desktop typography design guidelines `#c` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Typography](https://flathub.org/apps/org.gnome.design.Typography) - Tool for working with the GNOME desktop typography design guidelines `#c` `#gtk4` `#libadwaita`.
```

- [Gameeky](https://github.com/tchx84/gameeky) - Application to create and play games without any code for young learners and educators `#python` `#gtk4` `#libadwaita`.
- [Keypunch](https://github.com/bragefuglseth/keypunch) - Keyboard typing test like monkeytype `#rust` `#gtk4` `#libadwaita` `#gnome`.
- [Lutris](https://lutris.net) - Game launcher covering most gaming systems `#python` `#gtk3`.

> - [Maniatic Launcher](https://flathub.org/apps/io.github.santiagocezar.maniatic-launcher) - A launcher for the decompilation of Retro Engine v5, used in Sonic Mania Plus `#python` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Maniatic Launcher](https://flathub.org/apps/io.github.santiagocezar.maniatic-launcher) - Launcher for the decompilation of Sonic Manic Plus (Retro Engine v5) `#python` `#gtk4` `#libadwaita`.
```
